### PR TITLE
Add user invitation

### LINKED ISSUE #119: Add "User Invite" to Create User Flow
Dedicated button on Settings > Roles > Users ... "tags" style interface that is a CSV of email addresses that receive invites via email.

It would be nice to also have this flow integrated into the User Browse/Detail page too, perhaps as a dedicated "Invite User" action button in the header next to "New User".

### PR DESCRIPTION
**Pull request by Nitwel**
closes #119 
grafik
grafik

### CODE REVIEW

**Comment by benhaynes** [comment]
Perfect!! Thank you Nils...

Two questions:
* Let's ensure the proper label/field spacing is there... I think it's 8px, but we should confirm.
* Should this be a textarea instead of an input? Might make sense for longer CSVs

**Comment by Nitwel** [comment]
It is using the tags interface. So the button only gets clickable when you added a tag containing an email.
The default spacing for that is 4px as far as I know but I can look into that.

**Comment by benhaynes** [comment]
Confirmed that it's 8px (see below). Ahh, tags interface works!

Screen Shot 2020-10-20 at 10 00 58 AM

**Comment by Nitwel** [comment]
The `type-label` was a span and not a div. That's why it ignored that margin.

File: app/src/views/private/components/users-invite/users-invite.vue [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+			<v-card-text class="grid">
+				<div class="field">
+					<div class="type-label">{{ $t('emails') }}</div>
+					<interface-tags
+						v-model="emails"
+						:placeholder="$t('add_to_invite_user')"
+						icon-right="email"
+						whitespace=""
+					></interface-tags>
```
Comment (rijkvanzanten) [inline app/src/views/private/components/users-invite/users-invite.vue:14]:
```suggestion
					/>
```

File: app/src/views/private/components/users-invite/users-invite.vue [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+				</div>
+				<div class="field" v-if="role === null">
+					<div class="type-label">{{ $t('role') }}</div>
+					<v-select v-model="roleSelected" :items="roles"></v-select>
```
Comment (rijkvanzanten) [inline app/src/views/private/components/users-invite/users-invite.vue:18]:
```suggestion
					<v-select v-model="roleSelected" :items="roles" />
```

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]

File: app/src/views/private/components/users-invite/users-invite.vue [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+					persist: true,
+					type: 'error',
+				});
+			} finally {
+				loading.value = false;
+			}
+		}
+
+		async function loadRoles() {
+			const response = await api.get('/roles');
```
[the remark this answers is not in the document]
Comment (Nitwel) [reply app/src/views/private/components/users-invite/users-invite.vue:94]: How would I check for that?

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**